nautobot/core/tests/test_commands.py

```diff
@@ -25,6 +29,17 @@ def setUp(self):
         )
         self.git_repo.save(trigger_resync=False)
 
+        # Adding this test data to assert that https://github.com/nautobot/nautobot/issues/6081 is fixed
+        ct = ContentType.objects.get_for_model(GitRepository)
+        custom_field = CustomField.objects.create(type=CustomFieldTypeChoices.TYPE_TEXT, name="test_custom_field")
+        custom_field.content_types.add(ct)
+        obj_perm = ObjectPermission.objects.create(
+            name="Test permission",
+            constraints={"_custom_field_data__test_custom_field__in": ["test-1", "test-2"]},
+            actions=["view"],
+        )
+        obj_perm.object_types.add(ct)
```

Comment:
Should there be a test somewhere that checks for a warning to be logged for `Test permission`, independent of the `Test permission 2` test below?